fortuna/src/api/revelation.rs

```diff
@@ -29,6 +29,10 @@ use {
 ///
 /// Every blockchain supported by this service has a distinct sequence of random numbers and chain_id.
 /// Callers must pass the appropriate chain_id to ensure they fetch the correct random number.
+///
+/// We are registering the provider on chain with the following url:
+/// `base_uri/v1/chains/{chain_id}/revelations/`
```

Comment:
nit: `{base_uri}`

Reply:
Also, this comment doesn't make sense on this method. Someone looking to find out how the chain registration works isn't going to look at the code comment for a specific endpoint on the API.

Reply:
I think it would be better to put this code comment and the function get_register_uri in `api.rs`